src/content-helper/editor-sidebar/smart-linking/store.ts

```diff
@@ -20,13 +20,23 @@ export type SmartLinkingSettingsProps = {
 	maxLinksPerPost?: number;
 };
 
+/**
+ * Enum for the applyTo setting.
+ *
+ * @since 3.14.3
+ */
+export enum ApplyToOptions {
+	ALL = 'all',
+	SELECTED = 'selected',
```

Comment:
Unless there's a specific reason here, I'd suggest using the casing we use for other TypeScript enums (`All`, `Selected`).